.pullapprove.yml

```diff
@@ -2,20 +2,19 @@ version: 3
 
 # see https://docs.pullapprove.com/config/groups/
 groups:
-  # source code documentation change only: 2 approvals necessary
-  documentation:
+  documentation: documentation changes require 2 approvals
+    description:
     conditions:
-      - "'documentation' in labels"
+      - glob("docs/**/*.md") in files
```

Comment:
would change to docs/**

Reply:
I thought about this, but there is also code in `docs/` - that is, the Docsify code. But you're right, it should be possible to also change images etc. easily.